howdoi/howdoi.py

```diff
@@ -96,6 +96,8 @@ def _print_dbg(x): print("[DEBUG] " + x)  # noqa: E302
 CACHE_DIR = appdirs.user_cache_dir('howdoi')
 CACHE_ENTRY_MAX = 128
 
+SUPPORTED_HELP_QUERIES = ['use howdoi','howdoi']
```

Comment:
If I call the `howdoi` function passing only "howdoi" or "use howdoi" it works, but if I pass "howdoi howdoi" or "howdoi use howdoi" it doesn't (it fetches stack overflow like any other query). I'd say it's better to include these cases as well.

Reply:
> If I call the `howdoi` function passing only "howdoi" or "use howdoi" it works, but if I pass "howdoi howdoi" or "howdoi use howdoi" it doesn't (it fetches stack overflow like any other query). I'd say it's better to include these cases as well.

Hm. Makes sense. I’d just extend the queries supported to include these

Reply:
`howdoi run howdoi`
`howdoi do howdoi`

Reply:
There are just so many possible cases in terms of input commands intended to be query commands. For now, the commands I to the `supported_help_queries` are 
`['use howdoi','howdoi','run howdoi','do howdoi','howdoi howdoi','howdoi use howdoi']`